### PR TITLE
Pull request 1

### PR DESCRIPTION
**Pull request by AidanFahlman**
Added the initial README.md file. still trying to understand how this works :)

### CODE REVIEW

File: README.md
Comment (romancardenas) [inline README.md:30]: I find this requirement too technical. I suggest we remove it

File: README.md
Comment (romancardenas) [inline README.md:33]: Aren't points 7 to 9 just an extension of point 4?

File: README.md
Comment (romancardenas) [inline README.md:1]: We could change it to "Group E: Tool for supporting the simulation lifecycle" or something like that

File: README.md
Comment (romancardenas) [inline README.md:9]: Is it OK if we use the same text as the Deliverable 1?

File: README.md
Comment (romancardenas) [inline README.md:19]: Points 3 and 4 are like the final objective of this work. I would move them to the bottom of the list

File: README.md
Comment (romancardenas) [reply README.md:33]: Maybe we can move these requirements to the top of the list and remove the nested list of point 4

**Review by romancardenas — CHANGES_REQUESTED** [review]
I just checked the pull request. I added some comments that we should discuss and address before merging this to `devel` (by the way, I think that the pull request should be to `devel`, not the `main` branch

File: README.md
Comment (AidanFahlman) [reply README.md:9]: That is what I was considering doing but left it blank for the time being. Wanted to make sure everyone was ok with this

File: README.md
Comment (staubibr) [reply README.md:33]: I'm not sure they are exactly extensions of point 4. They will all require separate pieces of code. I thought they could be all presented as separate features. 

Point 4 is only the generation of the json file. 
Point 7 is the generation of the cpp files
Point 8 is the compilation
Point 9 is the execution

Maybe these could all be like sub-tasks of a main task. For those familiar with Agile terminology, this would be equivalent to an Epic with multiple Tasks.

File: README.md
Comment (staubibr) [reply README.md:9]: My bet is it will be ok. We'll probably hear more about this in the upcoming sync lectures.

File: README.md
Comment (staubibr) [reply README.md:19]: I have no strong feelings about this, I'm good with moving it to the end.

File: README.md
Comment (romancardenas) [reply README.md:33]: Nice. Maybe for the first deliverable, we can provide (for instance) 3 points, and the second deliverable would add the remaining point and integrate all the points.

File: README.md
Comment (romancardenas) [reply README.md:19]: Eventually, we'll have to sort the objective and give them a priority for the first deliverable

File: README.md
Comment (francoiseblanc) [reply README.md:1]: I agree with Román that our project name should be more descriptive 😊 Since our program will manage different tools, data, processes... we could also consider something like this: "Group E: A solution for simulation lifecycle management".

**Review by staubibr — DISMISSED** [review]

File: README.md
Comment (romancardenas) [reply README.md]: Bruno already addressed this. I'm resolving the conversation.

File: README.md
Comment (francoiseblanc) [reply README.md:45]: Are you referring to requirement 6? If so, I agree that we should remove it as it mainly refers to the implementation.

File: README.md
Comment (staubibr) [reply README.md:1]: I like Françoise's suggestion

File: README.md
Comment (staubibr) [reply README.md:45]: Just to clarify, the idea was that the system, in addition to offering spatial analysis capability, should also provide standard querying capabilities to build simulation models. Maybe the wording was not clear. But these are two different concepts in GIS. One relies on geometry the other, on descriptive attributes.

**Review by francoiseblanc — CHANGES_REQUESTED** [review]
Check new commited file and comments in the discussion.

File: README.md
Comment (staubibr) [inline README.md:47]: More of a comment than a change request. This list will really depend on the scenario we decide to tackle. So it should be updated to reflect that. Spatial Analysis is a large field and if we just say we'll do everything, we're in trouble. I'll try to bring this up Friday, maybe we can settle on a scenario then.

Comment (romancardenas) [reply README.md:47]: Perfect, less is more

**Review by staubibr — APPROVED** [review]

File: README.md
Comment (romancardenas) [reply README.md]: This issue has been addressed. I'm resolving the conversation

File: README.md
Comment (romancardenas) [reply README.md]: I'm resolving this conversation

File: README.md
Comment (romancardenas) [reply README.md]: This conversation has been addressed.

**Comment by romancardenas** [comment]
Hi, guys! I've just pushed my "final" review. I hope everyone is OK with it.

File: README.md
Comment (ghidadandan) [inline README.md:11]: Just some typos that I found.
In the first sentence of this paragraph: real-world (with a hyphen) 
In the last sentence: involved (instead of involve)

File: README.md
Comment (ghidadandan) [inline README.md:59]: The first sentence can be rephrased into this maybe?
> The purpose of the first release is to provide the necessary tools that enable users to upload all the required data related to the simulation lifecycle.

File: README.md
Comment (ghidadandan) [inline README.md:63]: integration of visualization **into** the simulation lifecycle

File: README.md
Comment (ghidadandan) [inline README.md:52]: Are we able to use the web service here since Bruno is working on it?

**Review by ghidadandan — APPROVED** [review]
It seems good to go if you overlook me acting like an English teacher 👍

File: README.md
Comment (staubibr) [reply README.md:52]: Yes. This is a different web service than the one I was talking about in the meeting.

Comment (ghidadandan) [reply README.md:52]: Sounds good!

**Review by francoiseblanc — APPROVED** [review]

**Review by romancardenas — APPROVED** [review]

**Pull request merged**
